EIPS/eip-7381.md

```diff
@@ -0,0 +1,378 @@
+---
+eip: 7381
```

Comment:
```suggestion
eip: 7409
```

Reply:
We needed to update the assets and the whole proposal as well, so I did it in the editor.